geode-core/src/distributedTest/java/org/apache/geode/internal/tcp/ConnectionCloseSSLTLSDUnitTest.java

```diff
@@ -141,7 +141,6 @@ public void beforeProcessMessage(final ClusterDistributionManager dm,
                         } catch (TimeoutException | InterruptedException e) {
                           fail("message observus interruptus");
```

Comment:
This is existing code, so just a quick comment here. This fail is invoked in a background thread which JUnit can't catch, so it'll typically fail during the grep for suspect strings but only if the product catches the `AssertionError` typically in a `catch Throwable` block with logging of the stack trace.

The better way to do this is use `DistributedErrorCollector` in a dunit test like this or `ErrorCollector` in all other types of tests:
```
@Rule
public DistributedErrorCollector errorCollector = new DistributedErrorCollector();
```
```
} catch (TimeoutException | InterruptedException e) {
  errorCollector.addError(e);
```
`DistributedErrorCollector` is useable from any thread in any DUnit VM. Basically each VM has its own `ErrorCollector` and then the tearDown of the Rule combines the errors collected within all of the VMs.

Reply:
Thank you for this reminder! I will change this `fail()`